File: 1-Iniciante/JavaScript/1012.js
let [A, B, C] = lines.shift().split(" ").map((x) => parseFloat(x));

const pi = 3.14159;

const triangulo = ((A * C) / 2);
const circulo = ((pi * Math.pow(C, 2)));
const trapezio = ((A + B) / 2) * C;
const quadrado = Math.pow(B, 2);
const retangulo = A * B;

console.log(`TRIANGULO: ${triangulo.toFixed(3)}`);
console.log(`CIRCULO: ${circulo.toFixed(3)}`);
console.log(`TRAPEZIO: ${trapezio.toFixed(3)}`);
console.log(`QUADRADO: ${quadrado.toFixed(3)}`);
console.log(`RETANGULO: ${retangulo.toFixed(3)}`);

File: 1-Iniciante/Java/1012.java
import java.io.IOException;
import java.util.Scanner;
 

public class Main {
 
    public static void main(String[] args) throws IOException {
        
       Scanner in = new Scanner(System.in);
        
        float A = in.nextFloat();
        float B = in.nextFloat();
        float C = in.nextFloat();
        float resultado = ((A*C)/2);
        System.out.printf("TRIANGULO: %.3f\n",resultado);
        float resultado2 = (float) ((3.14159)*(Math.pow(C, 2)));
        System.out.printf("CIRCULO: %.3f\n",resultado2);
        float resultado3 = (((A+B)/2)*C);
        System.out.printf("TRAPEZIO: %.3f\n",resultado3);
        float resultado4 = (float) Math.pow(B, 2);
        System.out.printf("QUADRADO: %.3f\n", resultado4);
        float resultado5 = A*B;
        System.out.printf("RETANGULO: %.3f\n", resultado5);
        
 
    }
 
}
